Dangerfile

```diff
@@ -97,10 +107,9 @@ end
 def checkForNpmInstallGlobal(file)
   # Warn developers that they are not supposed to use this flag
   fileDiff = git.diff_for_file(file)
-  # Ensure we keep using secure https:// references instead of http://
-  npmInstallMatches = fileDiff.patch.scan(/\+.*(npm install -g)|(npm install --global)/)
+  npmInstallMatches = fileDiff.patch.scan(/\+.*(npm install) (-g|--global)/)
```

Comment:
@eiditakahashi 
Sorry for not noticing this earlier, but are those parenthesis really needed?
`/\+.*(npm install) (-g|--global)/` => `/\+.*npm install (-g|--global)/`